pkgs/applications/graphics/json-plot/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoPackage,
+}: let
+  repo = "jp";
```

Comment:
Please inline with rec

Reply:
I cannot inline `repo`, as it lives inside `src`: please advise

Reply:
You should be able to access it via src.repo IIRC.